warehouse/models/mart/payments/_payments.yml

```diff
@@ -119,3 +119,54 @@ models:
       - name: transaction_date_pacific
       - name: day_of_week
         description: Day of week value of the transaction in the range [1,7] with Sunday as the first day of the week.
+  - name: fct_elavon__transactions
+    description: Transactions processed by Elavon
+    columns:
+      - name: payment_reference
+      - name: payment_date
+      - name: account_number
+      - name: routing_number
+      - name: fund_amt
+      - name: batch_reference
+      - name: batch_type
+      - name: customer_batch_reference
+      - name: customer_name
+      - name: merchant_number
+      - name: external_mid
+      - name: store_number
+      - name: chain
+      - name: batch_amt
+      - name: amount
+      - name: surchg_amount
+      - name: convnce_amt
+      - name: card_type
+      - name: charge_type
+      - name: charge_type_description
+      - name: card_plan
+      - name: card_no
+      - name: chk_num
+      - name: transaction_date
+      - name: settlement_date
+      - name: authorization_code
+      - name: chargeback_control_no
+      - name: roc_text
+      - name: trn_aci
+      - name: card_scheme_ref
+      - name: trn_ref_num
+        tests:
+          - unique
+      - name: settlement_method
+      - name: currency_code
+      - name: cb_acq_ref_id
```

Comment:
All of these with abbreviations (this and following) would be nice to explain more